assets/roks-metrics/roks-metrics-push-gateway-deployment.yaml

```diff
@@ -32,7 +32,7 @@ spec:
 {{- if .ROKSMetricsSecurityContext }}
 {{- $securityContext := .ROKSMetricsSecurityContext }}
         securityContext:
-          runAsNonRoot: {{ $securityContext.RunAsNonRoot }}
+          runAsUser: {{ $securityContext.RunAsUser }}
```

Comment:
Per our discussion, this should continue to use `runAsNonRoot` since we don't know the UID.